packages/webview_flutter/webview_flutter_wkwebview/ios/Classes/FLTWKNavigationDelegate.m

```diff
@@ -63,6 +63,18 @@ - (void)webView:(WKWebView *)webView
 }
 
 - (void)webView:(WKWebView *)webView didFinishNavigation:(WKNavigation *)navigation {
+  if (!self.shouldEnableZoom) {
+    NSString *source =
+        @"var meta = document.createElement('meta');"
+        @"meta.name = 'viewport';"
+        @"meta.content = 'width=device-width, initial-scale=1.0, maximum-scale=1.0,"
+        @"user-scalable=no';"
+        @"'width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no';"
```

Comment:
Looks like this got mangled in the relayout; you have a second copy of the string as its own statement.

Reply:
Fixed in 439e13b